src/flow/core/data.rs

```diff
@@ -225,6 +225,26 @@ impl<'a> From<&'a str> for DataType {
     }
 }
 
+impl fmt::Debug for DataType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match *self {
+            DataType::None => write!(f, "None"),
+            DataType::Text(..) => {
+                let text: Cow<str> = self.into();
+                write!(f, "Text ({:?})", text)
+            }
+            DataType::TinyText(..) => {
+                let text: Cow<str> = self.into();
+                write!(f, "TinyText ({:?})", text)
+            }
+            DataType::Int(n) => write!(f, "Int ({:?})", n),
+            DataType::BigInt(n) => write!(f, "BigInt ({:?})", n),
+            DataType::Timestamp(ts) => write!(f, "Timestamp ({:?})", ts),
+            DataType::Real(..) => write!(f, "Real ({})", self),
```

Comment:
Yeah, this seems fine. You might even want to use non-debug formatting for `Int`/`BigInt`, since their debug output is the same (afaik).

Reply:
👍